rst/topics/failover.rst

```diff
@@ -191,6 +217,13 @@ These are clusterwide parameters:
 * ``state_provider``: "tarantool" / "etcd".
 * ``tarantool_params``: ``{uri = "...", password = "..."}``.
 * ``etcd2_params``: ``{endpoints = {...}, prefix = "/", lock_delay = 10, username = "", password = ""}``.
+* ``fencing_enabled``: ``true`` / ``false``;
```

Comment:
I don't see any mentions of this option in code

Reply:
`fencing_enabled` emerged as a proposal during demonstration of the current fencing algorithm and it is not yet implemented. If you have any thoughts on this topic, please, feel free to share them [here](https://github.com/tarantool/cartridge/issues/950#issuecomment-721190021). Thank you!

Reply:
> it is not yet implemented

But why is this option documented? Would it be better to add this part after it will be implemented?

Reply:
It's true.  Will be done later: https://github.com/tarantool/cartridge/issues/1107
Removed from doc for now.

Reply:
What's about `failover_timeout `?

Reply:
Under consideration. Removed as well